### PR TITLE
ISSUE#102 - Automatically select "yes" for financial aid after clicking the finaid apply button

### PR DESCRIPTION
**Pull request by AbdurRafay01**
modified the _classes/register_ page when user visits it through financial aid link located at _/classes_.

### CODE REVIEW

File: src/pages/classes/register.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ import FinancialAidSubmissionSection from "../../components/classes/registration
 import PaymentSection from "../../components/classes/registration/PaymentSection"
 
 export default function ClassRegistrationPage() {
-  const [level, setLevel] = useState<"" | "beginner" | "intermediate">("")
+  const [level, setLevel] = useState<"" | "beginner" | "intermediate" | "fa">("")
```
Comment (maggie-j-liu) [inline src/pages/classes/register.tsx:19]: hmm, financial aid isn't a level, so you shouldn't set it here. i think just setting financialAid to true will work?

Comment (AbdurRafay01) [reply src/pages/classes/register.tsx:19]: Yeah, you are right, I'll remove the level part

File: src/pages/classes/register.tsx [hunk 2]
```diff
@@ -28,6 +28,9 @@ export default function ClassRegistrationPage() {
             : "intermediate"
         )
       }
+      else {
```
Comment (maggie-j-liu) [inline src/pages/classes/register.tsx:31]: probably have an explicit check for the hash being `fa`

Comment (maggie-j-liu) [reply src/pages/classes/register.tsx:31]: oh you need an explicit check otherwise financialAid is set to true all the time

**Review by maggie-j-liu — APPROVED** [review]

**Pull request merged**